### DISCUSSION TITLE
Add Metabolizing Enzyme...（Database not available）

### DISCUSSION THREAD

**Discussion by N-S-cbW** [Open-Systems-Pharmacology/Forum#837] [Questions & Problems] [answered]
How can I make database available?
I've downloaded the file **GENEDB_human.expressionDb**

**Comment by AndreDlm** [comment, marked as answer]
Did you specify the path to the gene expression database as described here? 
https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-options#template-database-path

**Comment by N-S-cbW** [comment]
> Did you specify the path to the gene expression database as described here?
> https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-options#template-database-path

Thank you, I didn't get it well bofore, the doc helps a lot.
